### PR TITLE
use storage-ng code to determine if "/" is on a network filesystem

### PR DESCRIPTION
**Pull request by wfeldt**
See also https://github.com/yast/yast-network/pull/529.

### CODE REVIEW

**Comment by mchf** [comment]
regarding new storage I've to believe you, however that one line looks good to me anyway. However since it is re-enabling a feature it deserves a version update in my POV.

File: src/modules/LanItems.rb [hunk 1]
```diff
@@ -2253,14 +2249,11 @@ def ProposeItem
       @ipaddr = ""
       @netmask = ""
       @bootproto = "dhcp"
-# storage-ng
-=begin
-      # #176804
-      if Storage.IsDeviceOnNetwork(NetworkStorage.getDevice("/")) != :no
+      devicegraph = Y2Storage::StorageManager.instance.staging
```
Comment (ancorgs) [inline src/modules/LanItems.rb]: Please, don't delete the original bug number `#176804`

**Comment by wfeldt** [comment]
This is for factory. They can't expect a bug number for everything we do there.

**Comment by ancorgs** [comment]
As @mchf said, I worry about the bnc/fate number. But other than that (we'll see if that's a problem), LGTM.

**Pull request merged**